prow/monitoring/prometheus_operator_crd/BUILD.bazel

```diff
@@ -0,0 +1,9 @@
+load("@io_bazel_rules_k8s//k8s:objects.bzl", "k8s_objects")
+
+k8s_objects(
+   name = "prometheus_operator_crds",
+   objects = [
+      "@operator_framework_community_operators//:upstream-community-operators/prometheus/alertmanager.crd.yaml",
```

Comment:
This is a string (and/or a path to a file), not a k8s_object target.


You'll need something like https://github.com/bazelbuild/rules_k8s#basic-deployment-objects 

You can use kind=None

Reply:
`$ bazel build //prow/monitoring/prometheus_operator_crd:prometheus_operator_crds` is working now. But I cannot make `apply` work.

```
$ bazel run //prow/monitoring/prometheus_operator_crd:prometheus_operator_crds.apply
ERROR: /home/hongkliu/go/src/k8s.io/test-infra/prow/monitoring/prometheus_operator_crd/BUILD.bazel:29:1: in objects attribute of _run_all rule //prow/monitoring/prometheus_operator_crd:prometheus_operator_crds.apply: target '//prow/monitoring/prometheus_operator_crd:alertmanager_crd.apply' does not exist. Since this rule was created by the macro 'k8s_objects', the error might have been caused by the macro implementation in /home/hongkliu/.cache/bazel/_bazel_hongkliu/ef5e8e918e6947b01dd429bce37b0e43/external/io_bazel_rules_k8s/k8s/objects.bzl:72:48
ERROR: /home/hongkliu/go/src/k8s.io/test-infra/prow/monitoring/prometheus_operator_crd/BUILD.bazel:29:1: in objects attribute of _run_all rule //prow/monitoring/prometheus_operator_crd:prometheus_operator_crds.apply: target '//prow/monitoring/prometheus_operator_crd:prometheus_crd.apply' does not exist. Since this rule was created by the macro 'k8s_objects', the error might have been caused by the macro implementation in /home/hongkliu/.cache/bazel/_bazel_hongkliu/ef5e8e918e6947b01dd429bce37b0e43/external/io_bazel_rules_k8s/k8s/objects.bzl:72:48
ERROR: /home/hongkliu/go/src/k8s.io/test-infra/prow/monitoring/prometheus_operator_crd/BUILD.bazel:29:1: in objects attribute of _run_all rule //prow/monitoring/prometheus_operator_crd:prometheus_operator_crds.apply: target '//prow/monitoring/prometheus_operator_crd:prometheusrule_crd.apply' does not exist. Since this rule was created by the macro 'k8s_objects', the error might have been caused by the macro implementation in /home/hongkliu/.cache/bazel/_bazel_hongkliu/ef5e8e918e6947b01dd429bce37b0e43/external/io_bazel_rules_k8s/k8s/objects.bzl:72:48
ERROR: /home/hongkliu/go/src/k8s.io/test-infra/prow/monitoring/prometheus_operator_crd/BUILD.bazel:29:1: in objects attribute of _run_all rule //prow/monitoring/prometheus_operator_crd:prometheus_operator_crds.apply: target '//prow/monitoring/prometheus_operator_crd:servicemonitor_crd.apply' does not exist. Since this rule was created by the macro 'k8s_objects', the error might have been caused by the macro implementation in /home/hongkliu/.cache/bazel/_bazel_hongkliu/ef5e8e918e6947b01dd429bce37b0e43/external/io_bazel_rules_k8s/k8s/objects.bzl:72:48
ERROR: Analysis of target '//prow/monitoring/prometheus_operator_crd:prometheus_operator_crds.apply' failed; build aborted: Analysis of target '//prow/monitoring/prometheus_operator_crd:prometheus_operator_crds.apply' failed; build aborted
INFO: Elapsed time: 0.116s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)

```

Reply:
Could it be caused by I do not have the permission on the current cluster?

Reply:
It got similar error on the CI below.